src/services/io/podio/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ plugin_glob_all(${PLUGIN_NAME})
 # but that never really worked quite right. Thus, we just force-run it
 # whenever cmake is run.
 execute_process (
-        COMMAND python3 ${PROJECT_SOURCE_DIR}/make_datamodel_glue.py WORKING_DIR=${PROJECT_BINARY_DIR} EDM4HEP_INCLUDE_DIR=${EDM4HEP_INCLUDE_DIR} EDM4EIC_INCLUDE_DIR=${EDM4EIC_INCLUDE_DIR}
+        COMMAND python3 ${PROJECT_SOURCE_DIR}/make_datamodel_glue.py WORKING_DIR=${PROJECT_SOURCE_DIR} EDM4HEP_INCLUDE_DIR=${EDM4HEP_INCLUDE_DIR} EDM4EIC_INCLUDE_DIR=${EDM4EIC_INCLUDE_DIR}
```

Comment:
What's the reason for this change? I'm wondering why re-building is now very slow, but reverting this change seems to help speed things up (only tested on one of my development branches).

Reply:
@nbrei in case he didn't get this (I created the PR so I may be the only one getting the notifications)

Reply:
It's because every factory now needs datamodel_glue.h for the PodioTypeMap that defines the relationship between podio types and their collections. The build dir for the podio plugin isn't on the include path for the other plugins, and the source dir is. I understand that using datamodel_glue all over the place is undesirable because it makes compile times longer and is just generally ugly. Luckily this is temporary. By adding one `using MyPodioObject::collection_t = MyPodioObjectCollection` to the generated EDM headers we can get rid of this.

Reply:
Contrary to this report, the compilation times don't change for me when I try to compile revisions before this PR.